README.rst

```diff
@@ -6,10 +6,29 @@ ftw.linkchecker
 Introduction
 ============
 
+``ftw.linkchecker`` is an add-on for Plone installations. It is meant to be run
+as a cronjob regularly to find broken links and references within Plone sites.
+
+How does it work
+****************
+
+- The config file is inspected and the information extracted.
+- Each Plone site is dealt with separately with its info extracted.
```

Comment:
with twice

Reply:
'For each Plone site (whether configured or not) in the zope instance ....' (then indent following lines)